office/zathura/Portfile

```diff
@@ -77,6 +77,11 @@ subport ${name}-docs {
     unset notes
 }
 
+variant synctex description "Enable synctex support" {
+    depends_lib-append  port:texlive-bin \
+                        port:dbus
+}
```

Comment:
What happens if those ports are already installed? Is there a configure setting to turn them off so they don't get used.

Reply:
If those ports are already installed, then the build system automatically detects it and enables synctex support.

There's no setting, so there's no way to turn them off AFAICT unless you want to patch meson.build to disable the part that checks if synctex is available.

Reply:
The problem is when it picks them up, they become a hidden dependency. So if `dbus` gets uninstalled, `zathura` won't prevent that because it is not a listed dependency (unless this variant is selected). And then the port could be in a broken state.

Reply:
Or does that scenario not break the port?

Reply:
It definitely breaks the port. I see two alternatives:

1. Patch the build system to disable the automatic check so that the default variant doesn't automatically pick up dbus/synctex, and then manually pass `-DWITH_SYNCTEX` when we want it
2. Fork out synctex into a separate subport from texlive-bin and make the default zathura installation depend on it. (This is what Debian does.)

Reply:
I think that option 1 is the simplest because the changes remain in this port and so that should be implemented. Option 2 requires coordination with the texlive maintainer - which certainly can be done in the future.

Reply:
I figured out how to make meson disable synctex without having to patch the meson.build file. I also dropped the dependency on dbus, because it turns out that it is a transitive dependency and that zathura does not directly link against it.